sdk/Lusid.Sdk.Tests/tutorials/Ibor/CutLabels.cs

```diff
@@ -0,0 +1,333 @@
+            var expectedLdnCloseInstrument2Holdings = _testDataUtilities.BuildPortfolioHolding(Currency, instrument2, (decimal)200.0, (decimal)20200.0);
+            var expectedLdnCloseInstrument3Holdings = _testDataUtilities.BuildPortfolioHolding(Currency, instrument3, (decimal)200.0, (decimal)19900.0);
+
+            // Retrieve holdings at LDNOpen today now that we have added a transaction (9am local time)
+            getHoldingsCutLabel = CutLabelFormatter(_currentDate, code["LDNOpen"]);
+            AssertHoldingsAtCutLabel(
+                getHoldingsCutLabel,
+                portfolioCode,
+                new List<PortfolioHolding>
+                {
+                    _testDataUtilities.BuildCashPortfolioHolding(Currency, currencyLuid, (decimal)90000.0),
+                    expectedLdnCloseInstrument1Holdings,
+                    expectedLdnOpenInstrument2Holdings,
+                    expectedLdnOpenInstrument3Holdings,
+                }
+            );
+
+            // Retrieve holdings at SGPClose today (10am local time)
+            getHoldingsCutLabel = CutLabelFormatter(_currentDate, code["SGPClose"]);
+            AssertHoldingsAtCutLabel(
+                getHoldingsCutLabel,
+                portfolioCode,
+                new List<PortfolioHolding>
+                {
+                    _testDataUtilities.BuildCashPortfolioHolding(Currency, currencyLuid, (decimal)80000.0),
+                    expectedLdnCloseInstrument1Holdings,
+                    expectedLdnCloseInstrument2Holdings,
+                    expectedLdnOpenInstrument3Holdings,
+                }
+            );
+
+            // Retrieve holdings at NYOpen today (2pm local time)
+            getHoldingsCutLabel = CutLabelFormatter(_currentDate, code["NYOpen"]);
+            AssertHoldingsAtCutLabel(
+                getHoldingsCutLabel,
+                portfolioCode,
+                new List<PortfolioHolding>
+                {
+                    expectedLdnCloseCashHoldings,
+                    expectedLdnCloseInstrument1Holdings,
+                    expectedLdnCloseInstrument2Holdings,
+                    expectedLdnCloseInstrument3Holdings,
+                }
+            );
+
+            // Retrieve holdings at LDNClose today (5pm local time)
+            // This will mean that the 4th transaction will not be included, demonstrating how cut labels work across time zones
+            getHoldingsCutLabel = CutLabelFormatter(_currentDate, code["LDNClose"]);
+            AssertHoldingsAtCutLabel(getHoldingsCutLabel, portfolioCode, new List<PortfolioHolding>
+                {
+                    expectedLdnCloseCashHoldings,
+                    expectedLdnCloseInstrument1Holdings,
+                    expectedLdnCloseInstrument2Holdings,
+                    expectedLdnCloseInstrument3Holdings,
+                });
+
+            _portfoliosApi.DeletePortfolio(TutorialScope, portfolioCode);
```

Comment:
i think this can me moved to the TearDown (if we make the portfolio code a class level var)

Reply:
Sure. Still trying to learn what the sensible patterns are for these things 😅